### PR TITLE
Update Windows.jsx

### PR DESCRIPTION
**Pull request by the-pro**
Wrote a patch of code to change the title as mentioned in (#653 ).

### CODE REVIEW

File: src/windows/Windows.jsx [hunk 1]
```diff
@@ -659,12 +659,20 @@ export default class WindowsIndex extends Component {
...
+        </DashboardPage>
+      );
+    }
+    if (bits === 64) {
+      return (
+        <DashboardPage title="Windows 64" subtitle="Release Criteria Report">
+          {reduced}
+        </DashboardPage>
+      );
+    }
```
Comment (klahnakoski) [inline src/windows/Windows.jsx]: Use the `bits` variable right tin the template, do not make `if/else` (or `if/if` in your case) blocks:

Example

    <DashboardPage title={"Windows "+ bits} ....

**Review by klahnakoski — CHANGES_REQUESTED** [review]

File: src/windows/Windows.jsx (hunk 1 above)
Comment (the-pro) [reply src/windows/Windows.jsx]: Done sir. Thanks for the review.

**Review by klahnakoski — CHANGES_REQUESTED** [review]
Almost done.

1. Run `yarn lint --fix`, and fix any errors it complains about
2. Run `yarn test -u` to update the test templates (which will capture the 64/32 changes)
3. Commit-and-push those changes
4. Ensure all the travis checks pass

**Review by klahnakoski — CHANGES_REQUESTED** [review]
@the-pro   This still does not pass tests; I do not know how the checks managed to miss that. Be sure to stage your changes then

    git commit -m "your message... 
    git push origin the-pro:patch-1 


image

**Comment by klahnakoski** [comment]
Sorry for the delay merging this.

**Pull request closed without merging**